packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1876,6 +1592,19 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 this.updateDocumentDirtyState(false);
             }
 
+            if (message.metadata?.compressed) {
+                this.logger.sendTelemetryEvent({
+                    eventName: "opToDecompress",
+                    message: JSON.stringify(message),
```

Comment:
You can't log message content like that - it will contain user data, which is a big NO.
Please log only metadata that is required to maintain the system.

Reply:
These telemetry events were just for local testing and aren't intended to actually go in with the change. Just makes it easier to see the result of the compression when running examples in the browser.